src/components/pomoclock/_pomodoro.scss

```diff
@@ -0,0 +1,70 @@
+.pomoclock-main {
+  width: 90vw;
+  max-width: 900px;
+  margin: 2rem auto;
+
+  .pomo-progress-bar {
+    width: 80%;
+    margin: 1rem auto;
+  }
+
+  .clock-action-btn {
+    width: 50%;
+    display: grid;
+    margin: 2rem auto;
+    grid-template-columns: 1fr 1fr;
+    gap: 1rem;
+    grid-template-rows: auto auto;
+    text-align: center;
+
+    & button:last-child {
+      grid-column: 1 / 3;
+    }
+  }
+
+  .info-message {
+    text-align: center;
+    font-size: 1.5rem;
+    color: var(--primary-color);
+    position: absolute;
+    top: 10%;
+    right: 1%;
+    background: black;
+    padding: 0.5rem;
```

Comment:
0.5rem may also be written in absolute units.

Reply:
Yes, we can do it in multiple ways. I prefer to use the `rem`